inference-engine/ie_bridges/python/sample/ngraph_function_creation_sample/README.md

```diff
@@ -1,54 +1,107 @@
-```sh
-./ngraph_function_creation_sample -h
-[ INFO ] InferenceEngine:
-        API version ............ <version>
-        Build .................. <number>
-        Description ....... API
-[ INFO ] Parsing input parameters
+Run the application with the -h option to see the usage message:
 
-ngraph_function_creation_sample [OPTION]
-Options:
+```
+python3 ngraph_function_creation_sample.py -h
+```
 
-    -h                      Print a usage message.
-    -m "<path>"             Path to a .bin file with weights for the trained model
-    -i "<path>"             Required. Path to an image or folder with images
-    -d "<device>"           Specify the target device to infer on it. See the list of available devices below. The sample looks for a suitable plugin for the specified device. The default value is CPU.
-    -nt "<integer>"         Number of top results. The default value is 10.
+Usage message:
 
-Available target devices: <devices>
+```
+usage: ngraph_function_creation_sample.py [-h] -m MODEL -i INPUT [INPUT ...]
+                                          [-d DEVICE] [--labels LABELS]
+                                          [-nt NUMBER_TOP]
 
+Options:
+  -h, --help            Show this help message and exit.
+  -m MODEL, --model MODEL
+                        Required. Path to a file with network weights.
+  -i INPUT [INPUT ...], --input INPUT [INPUT ...]
+                        Required. Path to an image file.
+  -d DEVICE, --device DEVICE
+                        Optional. Specify the target device to infer on; CPU,
+                        GPU, MYRIAD, HDDL or HETERO: is acceptable. The sample
+                        will look for a suitable plugin for device specified.
+                        Default value is CPU.
+  --labels LABELS       Optional. Path to a labels mapping file.
+  -nt NUMBER_TOP, --number_top NUMBER_TOP
+                        Optional. Number of top results.
 ```
 
-For example, to do inference of an UByte image on a GPU run the following command:
-```sh
-./ngraph_function_creation_sample -i <path_to_image> -m <path_to_weights_file> -d GPU
+To run the sample, you need specify a model weights and image:
+ - you can use images from the media files collection available at https://storage.openvinotoolkit.org/data/test_data.
+
+> **NOTE**: 
+> 
+> * This sample supports models with FP32 weights only.
+>
+> * The `lenet.bin` weights file was generated by the [Model Optimizer](../../../../../docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md) tool from the public LeNet model with the `--input_shape [64,1,28,28]` parameter specified.  
+>
+> * The original model is available in the [Caffe* repository](https://github.com/BVLC/caffe/tree/master/examples/mnist) on GitHub\*.
+
+You can do inference of an image using a pre-trained model on a GPU using the following command:
+
+```
+python3 ngraph_function_creation_sample.py -m <path_to_model>/alexnet.xml -i <path_to_image>/car.bmp -d GPU
```

Comment:
```suggestion
python3 ngraph_function_creation_sample.py -m <path_to_model>/lenet.bin -i <path_to_image>/car.bmp -d GPU
```

Reply:
python3 is absent on Windows, why we can't use python?